plutus-contract/src/Plutus/Contract/Request.hs

```diff
@@ -965,19 +973,6 @@ data MkTxLog =
         deriving stock (Show, Generic)
         deriving anyclass (ToJSON, FromJSON)
 
--- | Build a transaction that satisfies the constraints
-mkTxConstraints :: forall a w s e.
```

Comment:
The `Plutus.Contract.Request` is a public facing API module. We need to deprecate it.

Reply:
`mkTxConstraints` is not deleted, just moved. `mkTxContract` is deleted, but that had an awkward type and only exported under the "Etc." header, and is probably not used outside of our codebase.

Reply:
:+1: